Content/NPCs/Moonstone/Dreambeast.cs

```diff
@@ -54,11 +85,11 @@ public override void OnSpawn(IEntitySource source)
 
 				if (chain is null)
```

Comment:
OnSpawn is only for the spawner of the entity, which is the server only for NPCs. this will make the chains not exist for clients while making the server do verlet logic that it shouldn't